analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/NullPointerDereference.CSharp8.cs

```diff
@@ -22,6 +25,11 @@ public void NullCoalescenceOperator_Null_Noncompliant2()
             name = name ?? name.ToString(); // Noncompliant
         }
 
+        public void NullCoalesce_Conversion(Exception arg)  // Classes doesn't make sense here. They just demonstrate the flow.
```

Comment:
I don't understand the comment. Is it needed to understand the test.

Reply:
It's gone after the changes you've proposed. It was just weird to have `value` as disposable exception OR file stream created from the exception message